src/Winton.Extensions.Configuration.Consul/Extensions/KVPairQueryResultExtensions.cs

```diff
@@ -10,25 +10,24 @@
 
 namespace Winton.Extensions.Configuration.Consul.Extensions
 {
-    internal static class KVPairQueryResultExtensions
+  internal static class KVPairQueryResultExtensions
```

Comment:
I think the auto-formatting has also been applied here too.

Reply:
In fact, I think it's every file.

Reply:
rebased and fixed each commit one by one